accepted/exception-handling.md

```diff
@@ -0,0 +1,240 @@
+# Summary
+
+Implement support for the WebAssembly [exception handling
+proposal](https://github.com/WebAssembly/exception-handling).
+
+# Motivation
+[motivation]: #motivation
+
+The WebAssembly exception handling proposal introduces mechanisms for
+raising, catching, and propagating exceptions. The proposal has been
+subject to some controversy as the proposal has undergone several
+significant revisions, with an early revision being deployed in
+production already. As a result there effectively exists two revisions
+of exception handling in the wild at the moment:
+
+* The official W3C WebAssembly exception handling revised proposal;
+* and, the legacy exception handling revision.
+
+The revised proposal was advanced to phase 4 during [a community group
+meeting in
+July](https://github.com/WebAssembly/meetings/blob/main/main/2024/CG-07-16.md). Support
+for exception handling is (at least) of interest to C++, Kotlin, and
+OCaml toolchains. The proposal is also a prerequisite for the [stack
```

Comment:
```suggestion
for exception handling is (at least) of interest to C++, .NET, Kotlin, and
OCaml toolchains. The proposal is also a prerequisite for the [stack
```
The workaround we use in one of our toolchains to emulate exceptions has 10%+ code size cost (and probably even higher execution cost).